hasher-matcher-actioner/webapp/src/components/settings/ThreatExchangePrivacyGroupCard.tsx

```diff
@@ -63,110 +71,182 @@ export default function ThreatExchangePrivacyGroupCard({
+          <Card.Header className="mb-2 text-center">
+            {privacyGroupName}
           </Card.Header>
-          <Card.Subtitle className="mt-2 mb-2 text-muted">
-            Dataset ID: <CopyableTextField text={privacyGroupId} />
-          </Card.Subtitle>
-          <Card.Body className="text-left">
-            <Form>
-              <div>
-                <OverlayTrigger
-                  trigger="focus"
-                  placement="bottom"
-                  overlay={
-                    <Popover id={`popover-hashCount${privacyGroupId}`}>
-                      <Popover.Title as="h3">Hash Count</Popover.Title>
-                      <Popover.Content>{hashCount}</Popover.Content>
-                    </Popover>
-                  }>
-                  <Button variant="info">Hash Count</Button>
-                </OverlayTrigger>{' '}
-                <OverlayTrigger
-                  trigger="focus"
-                  placement="bottom"
-                  overlay={
-                    <Popover id={`popover-hashCount${privacyGroupId}`}>
-                      <Popover.Title as="h3">Match Count</Popover.Title>
-                      <Popover.Content>{matchCount}</Popover.Content>
-                    </Popover>
-                  }>
-                  <Button variant="info">Match Count</Button>
-                </OverlayTrigger>{' '}
-              </div>
-              <Form.Switch
-                onChange={onSwitchFetcherActive}
-                id={`fetcherActiveSwitch${privacyGroupId}`}
-                label="Fetcher Active"
-                checked={localFetcherActive}
-                disabled={!inUse}
-                style={{marginTop: 10}}
-              />
-              <Form.Switch
-                onChange={onSwitchMatcherActive}
-                id={`matcherSwitch${privacyGroupId}`}
-                label="Matcher Active"
-                checked={localMatcherActive}
-                disabled={!inUse}
-              />
-              <Form.Switch
-                onChange={onSwitchWriteBack}
-                id={`writeBackSwitch${privacyGroupId}`}
-                label="Writeback Seen"
-                checked={localWriteBack}
-                disabled={!inUse}
-              />
-            </Form>
-          </Card.Body>
+          <Card className="mx-2 mb-2">
+            <Card.Subtitle className="ml-3 my-2">
+              Dataset ID: <CopyableTextField text={privacyGroupId} />
+              <Accordion>
+                <Accordion.Toggle
+                  eventKey="0"
+                  as={Button}
+                  variant="outline-dark"
+                  size="sm"
+                  disabled={!description}
+                  onClick={() => setShowDescription(!showDescription)}
+                  className="my-2">
+                  Description
+                  <IonIcon
+                    className="inline-icon"
+                    icon={showDescription ? chevronDown : chevronUp}
```

Comment:
strong-nit: This threw me off while seeing your "After" video. 

Most cases I have seen, the chevron indicates what _will_ happen when I press the button. So it should point down when nothing is visible and point up when the description is visible.

I am sure it is possible to find references for the reverse behavior too, but here's what googling "should chevron point up or down" sent me to.

https://ux.stackexchange.com/questions/87350/which-way-should-arrows-point-for-a-dropdown-button

The top answer recommends down when not visible and up when visible.

Reply:
Agree, thanks for catching this